src/lib/rest/rest.cpp

```diff
@@ -265,6 +265,15 @@ static int uriArgumentGet(void* cbDataP, MHD_ValueKind kind, const char* ckey, c
 }
 
 
+/* ****************************************************************************
+*
+* uriParamGet -
+*/
+int uriParamGet(ConnectionInfo *ciP, const char* ckey, const char* val)
+{
+  return uriArgumentGet((void *) ciP, (MHD_ValueKind)0, ckey, val);
+}
+
```

Comment:
Fixed in c5a9b0a99a474ff8c3255c9a2d482f402c28bc27 and 61d094cc2ffa707f196ba056586d7ece6821d69b